### DISCUSSION TITLE
track your own website

### DISCUSSION THREAD

**Discussion by Plenium-ui** [YOURLS/YOURLS#3018] [Plugins]
Hi, i have a simply question: is it possible to use YOURLS for tracking my own website? Maybe, I thought I can implement a special script to my index.php and YORLS track / count the visitors and these visit time?

Thanks!!

**Comment by dgw** [comment]
You're better off just using Matomo or something else built for the purpose.

**Reply by ozh** [reply]
Using YOURLS to perform what Matomo or Google Analytics are built to do is like using a car as a replacement for an oven. Sure, with a lot of work, it could do.

**Reply by Plenium-ui** [reply]
No, not really. You think to big :). I just want to track, how many people visit my website. I can create a new shortet link for my website, but the long url is already known. Or I implement a shortet link to a hidden iframe on my website. So, I can track the visits. But these ways aren't really excellent. 
In my mind, I implement a little script or API in my code to track the visits ?!

**Reply by dgw** [reply]
Seriously, tools like Matomo are purpose-built for tracking website visits. YOURLS is a link shortener. You're asking us for tips on how to use your barbecue grill to manufacture stuffed animals—it doesn't make sense.

**Reply by Plenium-ui** [reply]
..but YOURLS also track the click rates...
